packages/discovery-provider/src/queries/query_helpers.py

```diff
@@ -377,9 +365,10 @@ def populate_user_metadata(
         user[response_name_constants.spl_wallet] = user_banks_dict.get(
             user["wallet"], None
         )
-        user[response_name_constants.does_follow_current_user] = (
-            user_id in follows_current_user_set
-        )
+
+        # this data is fetch in frontend via trpc.
+        # this field can probably be removed...
+        user[response_name_constants.does_follow_current_user] = False
```

Comment:
I can see why we're doing this, but leaving this and having it be false seems like it could introduce insidious bugs. Should we remove it entirely from the response? Or will that break expectations of that field existing on the frontend?

Reply:
Yeah that's a good call... client no longer uses this field so I removed it.